deploy/crds/saas.3scale.net_apicasts_crd.yaml

```diff
@@ -0,0 +1,353 @@
+                    failureThreshold:
+                      type: integer
+                      description: Override readiness probe failure failureThreshold
+                loadBalancer:
+                  type: object
+                  properties:
+                    proxyProtocol:
+                      type: string
+                      description: Proxy protocol enabled (k8s aws provider only accepts * by the moment)
+                    crossZoneLoadBalancingEnabled:
+                      type: boolean
+                      description: Enable (true) or disable (false) cross zone load balancing
+                    connectionDrainingEnabled:
+                      type: boolean
+                      description: Enable (true) or disable (false) connection draining
+                    connectionDrainingTimeout:
+                      type: integer
+                      description: Connection draining timeout (seconds)
+                    connectionHealthcheckHealthyThreshold:
+                      type: integer
+                      description: Connection healthcheck healthy threshold
+                    connectionHealthcheckUnhealthyThreshold:
+                      type: integer
+                      description: Connection healthcheck unhealthy threshold
+                    connectionHealthcheckInterval:
+                      type: integer
+                      description: Connection healthcheck interval (seconds)
+                    connectionHealthcheckTimeout:
+                      type: integer
+                      description: Connection healthcheck timeout (seconds)
+            staging:
+              type: object
+              required:
+                - image
+                - env
+                - marin3r
+                - externalDnsHostname
+              properties:
+                externalDnsHostname:
+                  type: string
+                  description: DNS hostname to manage on AWS Route53 by external-dns
+                image:
+                  type: object
+                  required:
+                    - tag
+                  properties:
+                    name:
+                      type: string
+                      description: Image name (docker repository)
+                    tag:
+                      type: string
+                      description: Image tag
+                    pullSecretName:
+                      type: string
+                      description: Quay pull secret for private repository
+                replicas:
+                  type: integer
+                  description: Number of replicas
+                env:
+                  type: object
+                  description: Environment variables
+                  required:
+                    - apicastConfigurationCache
+                    - threescalePortalEndpoint
+                  properties:
+                    apicastConfigurationCache:
+                      type: string
```

Comment:
Being a field representing time in seconds, shouldnt it be an integer, instead of a string? (applies to examples, docs, CRD)

Reply:
Validations added in 1d9096289bf609153e378ba6ae025774854cbc4f